_posts/2017/2017-10-06-epages-at-euruko.md

```diff
@@ -0,0 +1,49 @@
+
+Even thought Ruby is the central topic in most of them, it was nice to have other topics as empathy, system architecture or machine learning. For me that shows how mature and open minded is this community.
+
+## Lightning talks full of gems
+
+These talks were full of advices, from tips for juniors to system health monitoring. Some of the most interesting libraries for us were [tty](https://github.com/piotrmurach/tty) to ease the process of creating command line apps, [mini-check](https://github.com/workshare/mini-check) a small library to perform health checks or [reality](https://github.com/molybdenum-99/reality) an interesting experiment on ruby to process data and transform it to knowledge.
+
+## Euruko day two
+
+The second day started with a talk on [JRuby](http://jruby.org/) by [Charles Nutter](https://github.com/headius). It was awesome to see the issues, thought process and the numbers on other versions of Ruby and really intrigued by the performance and the good things that the JVM provides to Ruby, something that we are going to play with, for sure.
+
+This day, also had a really nice amount of different types of topics, not only focusing on the code. I like when people talk about health, life/work balance and how to manage stress; shows the human side of the programmers and how we often forget taking care about ourselves.
+
+Also we had some talks about things that went wrong, because again, we are humans, and we all make mistakes from time to time. This shows how important are other skills, like for example, being able to relax even in difficult situations to deliver the best quality and not snowballing mistakes.
+
+Talks ranting about Ruby where also present, showing again the open minded community and the high effort on improving the language; or about more scientific applications on ruby, to simulations of performance, cost efficiency and so on.
+
+## Things that caught our attention
+
+[Opal](https://github.com/opal/opal), a Ruby to JavaScript source-to-source compiler, [Rubymotion](http://www.rubymotion.com/) a way to write cross-platform native apps in Ruby or [jemalloc](https://github.com/jemalloc/jemalloc), were some of the technical highlights for us.
+
+On the non technical part, we were surprised by the openness of the community, openness to share, to contribute, to help; and by the passion of people about coding. All of this inspired probably by the motto of the Ruby community: "Matz is nice and so we are nice"
+
+It was also nice to see that not all Ruby is Rails, that, there are way too many uses for the language. I firmly believe that the sky is the limit for Ruby and sharing those two days with seven hundred people made clear that Ruby is not dying anytime soon.
```

Comment:
It was nice to see that not all Ruby is Rails and that there are way too many use cases for this language.